dev/tools/gen_defaults/lib/slider_template.dart

```diff
@@ -36,49 +36,87 @@ class _${blockName}DefaultsM3 extends SliderThemeData {
   Color? get disabledInactiveTrackColor => ${componentColor('$tokenGroup.disabled.inactive.track')};
 
   @override
-  Color? get disabledSecondaryActiveTrackColor => _colors.onSurface.withOpacity(0.12);
+  Color? get disabledSecondaryActiveTrackColor => ${componentColor('$tokenGroup.disabled.active.track')};
 
   @override
-  Color? get activeTickMarkColor => ${componentColor('$tokenGroup.with-tick-marks.active.container')};
+  Color? get activeTickMarkColor => _colors.${getToken("$tokenGroup.stop-indicator.color-selected")};
 
   @override
-  Color? get inactiveTickMarkColor => ${componentColor('$tokenGroup.with-tick-marks.inactive.container')};
+  // TODO(tahatesser): Update this hard-coded value to use the correct token value.
+  // https://github.com/flutter/flutter/issues/153271
```

Comment:
The fix https://github.com/flutter/flutter/pull/153722 is landed! Feel free to apply the updated tokens:)